src/matlab/phasemethods.cpp

```diff
@@ -196,6 +196,9 @@ void phasemethods(int nlhs, mxArray* plhs[],
         case 22:
             iok = thermo_getMolecularWeights(ph,nsp, &x[0]);
             break;
+        case 23:
+            iok = thermo_getSpeciesCharges(ph, nsp, &x[0]);
+            break;
```

Comment:
Are you planning on adding the `.m` file needed to make this available in Matlab?

Reply:
Well, you've already done the hard part by adding this and C wrapper in `ct.cpp`.

Reply:
Okay, I'm working on this now, but I've pushed the other change requests in the meantime.